styles.css

```diff
@@ -418,3 +426,469 @@ body {
     margin-bottom: 20px;
   }
 }
+
+@media only screen and (min-width: 768px) {
```

Comment:
- [ ] Please change the value of `min-width` any value between 992 and 1100 is acceptable because this is the desktop version, not the tablet version.
for example `min-width: 1080px`

Reply:
All good now 👍🏽